yt_dlp/extractor/huya.py

```diff
@@ -7,23 +7,25 @@
 from .common import InfoExtractor
 from ..utils import (
     ExtractorError,
+    clean_html,
     int_or_none,
     parse_duration,
     str_or_none,
     try_get,
     unescapeHTML,
-    unified_strdate,
+    update_url,
     update_url_query,
     url_or_none,
 )
 from ..utils.traversal import traverse_obj
 
 
 class HuyaLiveIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.|m\.)?huya\.com/(?!(?:video/play/))(?P<id>[^/#?&]+)(?:\D|$)'
     IE_NAME = 'huya:live'
-    IE_DESC = 'huya.com'
-    TESTS = [{
+    IE_DESC = '虎牙直播'
+
+    _VALID_URL = r'https?://(?:(?:m|www)\.)?huya\.com/(?!(?:video/play/))(?P<id>[^/#?&]+)(?:\D|$)'
```

Comment:
please don't unnecessarily move the line when you are changing it; it makes eyeballing the diff more difficult when reviewing. please revert the added newline also